easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-4.5.5-foss-2017a-ParMETIS-4.0.3.eb

```diff
@@ -0,0 +1,20 @@
+name = 'SuiteSparse'
+version = '4.5.5'
+
+homepage = 'http://faculty.cse.tamu.edu/davis/suitesparse.html'
+description = """SuiteSparse is a collection of libraries manipulate sparse matrices."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+toolchainopts = {'unroll': True, 'pic': True}
+
+source_urls = ['http://faculty.cse.tamu.edu/davis/SuiteSparse/']
+sources = [SOURCE_TAR_GZ]
+
+parmetis = 'ParMETIS'
+parmetis_ver = '4.0.3'
+versionsuffix = '-%s-%s' % (parmetis, parmetis_ver)
```

Comment:
same here, no variable for `'ParMETIS'`, keep `versionsuffix` close to `version`